docs/user_docs/cli/kbcli_cluster_create_mysql.md

```diff
@@ -16,26 +16,36 @@ kbcli cluster create mysql NAME [flags]
   
   # Create a cluster with the specified cpu, memory and storage
   kbcli cluster create mysql --cpu 1 --memory 2 --storage 10
+  
+  # Create a cluster with smartengine
+  kbcli cluster create mysql --smart-engine-enabled
+  
+  # Create a cluster with specified user customized config template
+  kbcli cluster create mysql --userConfigTemplate.templateRef=mysql8.0-config-template --userConfigTemplate.namespace=kb-system --userConfigTemplate.policy=none
```

Comment:
Using `--userConfigTemplate.templateRef` as a flag for the command looks a bit strange. The naming as a combination of camelCase and . is inconsistent with the all-lowercase format connected by hyphen of kbcli command line.

I suggest to use `config-template-name`, `config-template-namespace` and `config-template-policy` to replace them. I think the 'user' is a bit redundant, as there is no other field that could be confused with config-template in this command. Therefore, I think using just 'config-template' as prefix would be sufficient.

Additionally, I noticed that in the `kbcli cluster configure` sub-command, the flag used to represent the configuration template meaning is `config-spec`, could we keep it consistent and all use config-template?

```
    --config-spec='':
        Specify the name of the configuration template to be updated (e.g. for apecloud-mysql:
        --config-spec=mysql-3node-tpl). For available templates and configs, refer to: 'kbcli cluster
        describe-config'.

```